src/genie/libs/parser/iosxr/show_mpls.py

```diff
@@ -71,6 +71,217 @@ def cli (self, output=None):
             
         return mpls_dict
 
+# ==============================================
+# Parser for 'show mpls ldp discovery'
+# ==============================================
+class ShowMplsLdpDiscoverySchema(MetaParser):
+
+    """Schema for show mpls ldp discovery
+                  show mpls ldp discovery detail
+                  show mpls ldp afi-all discovery
+                  show mpls ldp discovery <ldp>
+                  show mpls ldp vrf <vrf> discovery
+                  show mpls ldp vrf <vrf> discovery detail
+    """
+
+    schema = {
```

Comment:
as same as `iosxe`, better to cover the output of discovery for targeted hellos.
https://github.com/CiscoTestAutomation/genieparser/blob/master/src/genie/libs/parser/iosxe/show_mpls.py#L1174
Here is sample output. Can I ask you to add the schema and unittest?
```
RP/0/0/CPU0:iosxrv-1#show mpls ldp vrf default discovery detail 
Fri Jan  1 13:13:12.527 UTC

Local LDP Identifier: 10.1.1.1:0
Discovery Sources:
  Interfaces:
    GigabitEthernet0/0/0/0 (0x40) : xmit/recv
      VRF: 'default' (0x60000000)
      Source address: 10.1.2.1; Transport address: 10.1.1.1
      Hello interval: 5 sec (due in 3.7 sec)
      Quick-start: Enabled
      LDP Id: 10.2.2.2:0
          Source address: 10.1.2.2; Transport address: 10.2.2.2
          Hold time: 15 sec (local:15 sec, peer:15 sec)
                     (expiring in 13.6 sec)
          Established: Jan  1 12:37:06.766 (00:36:05 ago)

  Targeted Hellos:
    10.1.1.1 -> 10.3.3.3 (active), xmit/recv
      Hello interval: 6 sec (due in 4.2 sec)
      Quick-start: Enabled
      LDP Id: 10.3.3.3:0
          Hold time: 60 sec (local:60 sec, peer:60 sec)
                     (expiring in 55.3 sec)
          Established: Jan  1 13:11:39.034 (00:01:33 ago)

Reply:
I've added 'Targeted Hellos:' to the schema but unlike iosxe, 'Hello interval', 'quick-start', 'hold time', 'established' are also used for interfaces, hence I don't know how to manage  it.

Reply:
hmm. ok. if so, let's remove the `targeted hellos` from schema now. we can add when it's needed. sorry for bothering you...

Reply:
No problem 👍 , I've commented the targeted hellos part.